tools/run_tests/xds_k8s_test_driver/framework/infrastructure/traffic_director.py

```diff
@@ -290,6 +292,26 @@ def delete_forwarding_rule(self, force=False):
         self.compute.delete_forwarding_rule(name)
         self.forwarding_rule = None
 
+    def create_firewall_rule(self, force=False):
+        name = self._ns_name(self.FIREWALL_RULE_NAME)
+        logging.info('Creating firewall rule "%s" in network "%s"', name,
+                     self.network)
+        resource = self.compute.create_firewall_rule(
+            name, self.network_url, ['35.191.0.0/16', '130.211.0.0/22'],
+            ['8080-8100'])
```

Comment:
This should be an argument `healthcheck_port`.

Reply:
Done. Along with the other comment.